runtime/pendulum/src/lib.rs

```diff
@@ -6,16 +6,15 @@
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
-mod currency;
 mod weights;
 pub mod xcm_config;
 pub mod zenlink;
 use crate::zenlink::*;
 use xcm::v3::MultiLocation;
 use zenlink_protocol::{AssetBalance, MultiAssetsHandler, PairInfo};
 
-pub use currency::CurrencyId as PendulumCurrencyId;
 pub use parachain_staking::InflationInfo;
+pub use spacewalk_primitives::CurrencyId as PendulumCurrencyId;
```

Comment:
Nit: I think I would prefer if we don't export this as `PendulumCurrencyId` anymore. I checked the other references and it's only referenced in the integration tests. It's kind of redundant info, since this type is declared/exported in the pendulum runtime. In the related files of the integration tests it's imported as `pendulum_runtime::PendulumCurrencyId` but I think `pendulum_runtime::CurrencyId` would be just as easy to understand.

Reply:
Thanks for the comment @ebma

`pendulum_runtime::PendulumCurrencyId` was removed. I've changed the `pendulum_runtime::PendulumCurrencyId` for `pendulum_runtime::CurrencyId` in the integration tests.